activerecord/test/cases/log_subscriber_test.rb

```diff
@@ -169,6 +170,27 @@ def test_exists_query_logging
     assert_match(/SELECT .*?FROM .?developers.?/i, @logger.logged(:debug).last)
   end
 
+  def test_vebose_query_logs
+    ActiveRecord::Base.verbose_query_logs = true
+
+    event = Struct.new(:duration, :payload)
```

Comment:
Can you change these tests to use the new `Event` struct class, to avoid duplication?